src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/PrintReadsSparkIntegrationTest.java

```diff
@@ -157,11 +127,9 @@ public void testSequenceDictionaryValidation() throws Exception {
                 {"print_reads.sorted.queryname.bam", ".bam", null},
                 {"print_reads.sorted.queryname.cram", ".sam", "print_reads.fasta"},
                 {"print_reads.sorted.queryname.cram", ".bam", "print_reads.fasta"},
-
-                //comparing queryname sorted cram files blows up: https://github.com/broadinstitute/gatk/issues/1271
-//                {"print_reads.sorted.queryname.cram", ".cram", "print_reads.fasta"},
-//                {"print_reads.sorted.queryname.sam", ".cram", "print_reads.fasta"},
-//                {"print_reads.sorted.queryname.bam", ".cram", "print_reads.fasta"}
+                {"print_reads.sorted.queryname.cram", ".cram", "print_reads.fasta"},
+                {"print_reads.sorted.queryname.sam", ".cram", "print_reads.fasta"},
+                {"print_reads.sorted.queryname.bam", ".cram", "print_reads.fasta"}
```

Comment:
These tests never get as far as file comparison; they pass because the test using this data provider has an expectedException because ReadSparkSink currently only handles coordinate sorted output.